scopes/react/react/webpack/webpack.config.preview.ts

```diff
@@ -206,7 +206,6 @@ export default function (fileMapPath: string): Configuration {
         '@teambit/ui.mdx-scope-context': require.resolve('@teambit/ui.mdx-scope-context'),
         'react-dom/server': require.resolve('react-dom/server'),
         'react-dom': require.resolve('react-dom'),
-        'react-native': 'react-native-web',
```

Comment:
why did you remove it? it's required

Reply:
maybe it's because it's in the react webpack and not in the react-native webpack?